### ISSUE TITLE
Google Fonts Support

### ISSUE THREAD

**Issue by subframe7536** [subframe7536/maple-font#272]
I plan to upload this font to Google Fonts in next major version.

I'm using Fontlab, and its generated `.glyphs` / `.glyphspackage` will emit error while building using `fontmake`

```
fontmake: Error: In 'source\\MapleMono[wght].glyphspackage' -> 'master_ufo\\MapleMono.designspace': Compiling UFO failed: mark anchor cannot be numbered: '_t.uni031B_2'
```

My command: 

```sh
fontmake -g ./source/MapleMono[wght].glyphspackage --keep-overlap
```

Seems related issue: https://github.com/googlefonts/fontmake/issues/1104

**Comment by subframe7536** [comment]
Run with `fontbakery` and fail on some rules.

- `family/win_ascent_and_descent`: some box symbols in exceed winAscent and winDescent. I just copy them from JetBrains Mono

**Comment by subframe7536** [comment]
Find out how to fix the broken `fontmake` build: migrate correct anchors from JetBrains Mono.

**Comment by mogeko** [comment]
I'm glad to see progress on this issue!

I'm just curious which version of Maple Mono will be uploaded to Google Fonts? Maple Mono or Maple Mono CN?

**Comment by subframe7536** [comment]
Indeed, I'm not sure, but I don't think the CN part is a good font due to its glyph quality.

<img src="https://github.com/user-attachments/assets/3d2bf778-c59e-47cf-8321-7c31b1d4cb57" width="400" />

**Comment by subframe7536** [comment]
https://github.com/google/fonts/issues/9244

**Comment by mogeko** [comment]
I really admire your pursuit of aesthetics!

However, in reality, there has been a long-term lack of a monospaced Chinese font that is friendly to use on Google Fonts. Even Noto Mono CJK is not available on Google Fonts.💔

It would be perfect if both Maple Mono and Maple Mono CN could be provided.

**Comment by subframe7536** [comment]
@codex Please optimize the current V8 (`variable` branch) for Google Fonts onboarding and prepare a focused PR.

Context:
- Upstream onboarding issue: google/fonts#9244 (`Add Maple Mono`), currently accepted but iceboxed pending CJK expertise.
- Current V8 sources/build pipeline have changed substantially since the older GF preparation work in #272 / #395, so audit the current branch instead of reviving old patches blindly.
- Preserve all existing Maple Mono release/custom-build behavior. Do not redesign CJK glyphs or change Nerd Font packaging in this PR.

Primary goal: make the canonical Maple Mono Latin variable family reproducibly buildable and QA-able according to the current Google Fonts requirements, while keeping the existing CJK/NF variants working.

Required work / acceptance criteria:

1. Fix embedding permissions at the source level.
   - All relevant Latin UFO masters currently contain `openTypeOS2Type` with bit 3; Google Fonts requires final `OS/2.fsType == 0` (Installable embedding).
   - Remove/correct the UFO setting and add a regression check that inspects generated TTFs.

2. Add a Google Fonts-specific build path instead of changing normal release defaults.
   - Reuse the existing Python pipeline as much as possible.
   - Build canonical `Maple Mono` Regular and Italic variable TTFs only for the GF path.
   - No Nerd Font glyph injection, no CJK merge, no OTF/WOFF2 requirement for this path.
   - Variable fonts must be unhinted (or passed through the current recommended `gftools fix-nonhinting` flow).
   - Keep the regular/default instance at wght 400 and preserve the current intended weight mapping/designspace behavior.

3. Bring the upstream repository layout into compliance with the current GF upstream guide with the smallest maintainable migration.
   - Canonical design sources must live under `sources/` (currently `source/`). Update all build scripts, tests, workflows and documentation references atomically.
   - Provide `sources/build.sh` and/or a supported `sources/config.yaml` so GF binaries can be produced in one command.
   - Add `CONTRIBUTORS.txt` without inventing contributor identities or email addresses; use verifiable existing repository data only.
   - Provide/normalize the required `documentation/` and `fonts/` structure. If moving `docs/` is necessary, update every internal link/reference. Additional project-specific dirs such as `scripts/` and `resources/` may remain if useful.
   - Do not commit unrelated release artifacts.

4. Add Google Fonts QA to CI.
   - Add/pin the appropriate `gftools` / `fontbakery` dependencies in the project’s existing uv/pyproject workflow; regenerate `uv.lock` and exported `requirements.txt` consistently.
   - Add a dedicated GitHub Actions job/workflow that runs the GF build and `fontbakery check-googlefonts` on the generated canonical variable TTFs.
   - Also verify at minimum: `fsType=0`, `post.isFixedPitch=1`, appropriate OS/2 PANOSE monospace proportion, copyright/license strings, weight axis includes 400, valid STAT/fvar/name data, vertical metrics, and GF Latin Core coverage.
   - Fix actionable FontBakery FAILs found during this work. Do not add FontBakery exceptions merely to silence failures unless the exception is genuinely required and is documented in the PR.

5. Preserve existing project behavior.
   - `python build.py` / current default config and custom/NF/CJK release flows must keep working as before, except for standards-correct metadata fixes such as fsType.
   - Update tests for any path/config migration.
   - Update README/build docs only where needed to explain the GF build/QA workflow.

6. Validation before finishing:
   - Run the existing formatting/lint/type/unit/dry-run checks from `.github/workflows/python.yml`.
   - Run the new GF one-command build.
   - Run `fontbakery check-googlefonts` and summarize PASS/WARN/FAIL counts in the PR description.
   - Inspect the final TTF tables programmatically (fontTools is fine) and report the key metadata values.

Implementation guidance:
- Prefer a small GF-specific profile/entrypoint over contaminating the normal `config.json` defaults.
- Do not make speculative glyph-design changes, especially to CJK.
- Keep generated file names compatible with Google Fonts variable-font naming conventions.
- Treat current official GF documentation as source of truth if old #272/#395 discussion conflicts with it.

Please create a PR against `variable` with a clear summary of changes, remaining FontBakery warnings (if any), and any onboarding blockers that require a human/CJK reviewer.
